quickstart/docker-compose.yml

```
@@ -62,6 +62,13 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
     command: sh -c "((sleep 15 && echo 'kafka up' && kafka-topics.sh --create --if-not-exists --bootstrap-server 127.0.0.1:9092 --replication-factor 1 --partitions 3 --topic consumer)&) && /opt/bitnami/scripts/kafka/run.sh"
 
+  kafka-exporter:
+    image: bitnami/kafka-exporter
+    command: [ "--kafka.server=127.0.0.1:9092" ]
+    network_mode: host
+    expose:
+      - 9308
```

```suggestion
    healthcheck:
      test:
        [
          "CMD-SHELL",
          "kafka-topics.sh --bootstrap-server 127.0.0.1:9092 --topic consumer --describe"
        ]

  kafka-exporter:
    image: bitnami/kafka-exporter
    container_name: kafkaexporter
    command: [ "--kafka.server=127.0.0.1:9092" ]
    network_mode: host
    expose:
      - 9308
    depends_on:
      kafka:
        condition: service_healthy
```

because exporter complains that kafka is not available